grafana/grafana
# loki: rename result_transformer* to live_streams_result_transformer*

in the Loki data source, with the backend-migration happening, the only frontend-mode-code remaining is the one doing the live-streaming.

it will take some time to migrate that part to backend-mode, so i did a small code-cleanup PR for the current live-streaming code. the `result_transformer.ts` file is now only used by the live-streaming-code, so i renamed it to `live_streaming_result_transformer.ts`, and adjusted the related files.

how to test:
- `make devenv sources=loki`
- go to explore-mode, choose loki datasource, run this query `{place="moon"}`
- afterwards press the live-streaming-button in the upper-right corner
- you should see log-lines appear continuously
